Makefile

```diff
@@ -1,11 +1,12 @@
 PROJECT_ROOT=github.com/uber/jaeger-lib
-PACKAGES := $(shell glide novendor | grep -v ./thrift-gen/...)
+PACKAGES := $(shell go list ./... | grep -v '^github.com/uber/jaeger-lib$$' | sed 's|github.com/uber/jaeger-lib/||g' | sed 's|/.*$$||g' | sort -u | sed 's|^|./|g' | sed 's|$$|/...|g')
```

Comment:
`go list ./... | awk -F\/ 'NR>1 {print "./"$4"/..."}' | sort -u` has the same effect. I find it more readable, but feel free to keep the chained sed if you think it's clearer.

Reply:
nice!  Sadly, I never learned awk to the point of day-to-day usefulness.